readme.md

```diff
@@ -1,6 +1,13 @@
-1. We strive to create a welcoming and inclusive community for everyone, regardless of their race, ethnicity, gender, sexual orientation, religion, or any other personal identity.
-2. Bigotry, hate speech, harassment, or any other form fo discrimination will not be tolerated and will result in an immediate ban.
-3. Keep conversations in the appropriate channels. 
-4. No spamming
-5. Do not share personal information
-6. If you witness any rule violations or feel unsafe, please let a moderator know.
+Before joining our community, please take a moment to review and accept the following guidelines. We designed these rules to ensure Duduke provides a safe, respectful, and welcoming environment for all members, regardless of their race, background, or other personal characteristics.
```

Comment:
Only a list of rules can be set up in Discord's "Rule Screening" page. So everything should be a list item.

Reply:
Done.